### ISSUE TITLE
Applying filter to the video

### ISSUE THREAD

**Issue by aloklearning** [bhuvnesh123/FFmpeg-Video-Editor-Android#10]
I've followed your link and got enough information on how to use the FFMPEG. I'm using FFMPEG to apply filter to the specified file. However the FFMPEG is not working fine. This is the link where I've asked the question, please help so that I'll reach to the desired location.

[https://stackoverflow.com/questions/45978324/ffmpeg-filter-is-not-implementing](url)

**Comment by aloklearning** [comment]
Why the output is coming in rotated form : 
 I've used this and I'm getting the output as 90 degree clockwise rotated, is there any way to do that. Have used the **transpose=2** to rotate it 90 degree counterclockwise inside the command, it is then rotating the video only not applying the filter. Any idea. The command I've used is  : 

`String complexCommand[] = {"-y", "-i", stringBuilder.toString(), "-strict", "experimental", "-vf", "curves=vintage",
                "-vf", "transpose=2", "-s", "640x480", "-r", "30", "-aspect", "4:3", "-ab",
                "48000","-ac", "2", "-ar", "22050", "-b:a", "2097k", "-vcodec", "mpeg4", savingPath};`

I've used **"-vf", "curves=vintage"** only then the output was coming in the filtered form but roated so I thot of doing like this.
